client/elements/addons/sc-top-sheet-views.js

```diff
@@ -330,6 +338,31 @@ export class SCTopSheetViews extends LitLocalized(LitElement) {
     super.connectedCallback();
     this._loadScToasts();
     this._fetchReferenceDisplayType();
+    document.addEventListener('keydown', this._handleKeydown.bind(this));
+    window.setTimeout(() => {
+      this.show();
+      this.hide();
+    }, 2000);
```

Comment:
This timeout is absolutely necessary to make it work, but I can't for the life of me figure out why.  Without this `show/hide`, the toast pops up showing that the key press was registered, but the layout doesn't actually change.  Any ideas, @ihongda ?

Reply:
Thanks for playing with it @thesunshade ! Your bug is certainly the same one this is attempting to work around. It will definitely need a real fix before this can merge. That's waiting on @ihongda's knowledge: what gets hooked up only after the top panel shows on a given sutta?

Reply:
> This timeout is absolutely necessary to make it work, but I can't for the life of me figure out why. Without this `show/hide`, the toast pops up showing that the key press was registered, but the layout doesn't actually change. Any ideas, @ihongda ?

Bhante 🙏🏼 

This part of the code can be placed in the `firstUpdated` event of lit.
>Called after the component's DOM has been updated the first time, immediately before [updated()](https://lit.dev/docs/components/lifecycle/#updated) is called.

In the firstUpdated event, we should be able to remove the `setTimeout`.

Reply:
What are you talking about @ihongda? You mean put the addEventListener to the firstUpdated callback instead of the connectedCallback?  Cause that doesn't work.  I'm telling you that there's something that's only getting hooked up after the top sheet actually _opens_ (firstUpdated is called before that).